CMakeLists.txt

```diff
@@ -39,10 +39,10 @@ set_property(DIRECTORY APPEND PROPERTY ADDITIONAL_MAKE_CLEAN_FILES .rocks)
 add_custom_command(
   OUTPUT ${PROJECT_SOURCE_DIR}/.rocks
   DEPENDS ${PROJECT_NAME}-scm-1.rockspec
-  COMMAND tarantoolctl rocks make ./${PROJECT_NAME}-scm-1.rockspec
-  COMMAND tarantoolctl rocks install http 1.1.0
-  COMMAND tarantoolctl rocks install luacheck 0.25.0
-  COMMAND tarantoolctl rocks install luacov 0.13.0
+  COMMAND tt rocks make ./${PROJECT_NAME}-scm-1.rockspec
```

Comment:
Install `tt` via `apt-get -y install tt` before building the project.

Reply:
Fixed.